tests/codegen/checked_ilog.rs

```diff
@@ -1,11 +1,23 @@
 //@ compile-flags: -O
+//@ revisions: others riscv64 loongarch64
+
+//@[others] ignore-riscv64
+//@[others] ignore-loongarch64
+//@[riscv64] only-riscv64
+//@[riscv64] compile-flags: --target riscv64gc-unknown-linux-gnu
+//@[riscv64] needs-llvm-components: riscv
+//@[loongarch64] only-loongarch64
+//@[loongarch64] compile-flags: --target loongarch64-unknown-linux-gnu
+//@[loongarch64] needs-llvm-components: loongarch
 
 #![crate_type = "lib"]
 
 // Ensure that when val < base, we do not divide or multiply.
 
 // CHECK-LABEL: @checked_ilog
-// CHECK-SAME: (i16 noundef %val, i16 noundef %base)
+// others-SAME:      (i16 noundef %val, i16 noundef %base)
+// riscv64-SAME:     (i16 noundef zeroext %val, i16 noundef zeroext %base)
+// loongarch64-SAME: (i16 noundef zeroext %val, i16 noundef zeroext %base)
```

Comment:
For tests where checking the zeroext/signext attributes is not the point of the test, I'd prefer to just use a `{{.*}}` wildcard, rather than splitting this by architecture.

Reply:
like so?
```suggestion
// CHECK-SAME: (i16 noundef {{.*}} %val, i16 noundef {{.*}} %base)
```